apps/client/src/plugins/Measurer/MeasurerView.js

```diff
@@ -25,12 +26,6 @@ const StyledToggleButton = styled(ToggleButton)(({ theme }) => ({
   img: {
     filter: theme.palette.mode === "dark" ? "invert(1)" : "",
   },
-  "&.Mui-selected, &.Mui-selected:hover": {
```

Comment:
Why was this removed?

Reply:
Think cos we use tooltip in Measurer instead